README.md

```diff
@@ -1 +1,19 @@
 # hw01
+# Biography: Madison Martin 
+
+I usually go by Madison in school, but I also go by *Maddie* as well. 
+I like both, so feel free to call me whichever is easiest! 
+I am a MAPSS student pursuing the Computational Social Science certificate, which is how I found this course. 
+My research interests are prejudice and discrimination, based on race/ethnicity or disability, in organizational contexts like the workplace or in education. 
+During my time in this course, and over the MAPSS program in general, I am hoping to: 
+
+1. gain coding skills by being introduced to the R programming language
+
+2. utilize these new programming-focused skills in the process of creating my master's thesis project 
+
+3. expand my knowledge and understanding of the role data plays in social science
```

Comment:
The list turned out great!

Reply:
Note you don't have to manually itemize the list. Markdown does this automatically

```
1. gain coding skills by being introduced to the R programming language
1. utilize these new programming-focused skills in the process of creating my master's thesis project 
1. expand my knowledge and understanding of the role data plays in social science
```

becomes

1. gain coding skills by being introduced to the R programming language
1. utilize these new programming-focused skills in the process of creating my master's thesis project 
1. expand my knowledge and understanding of the role data plays in social science